compiler/src/diagnostics/lenses.re

```diff
@@ -2,76 +2,154 @@ open Grain_typed;
 
 [@deriving yojson]
 type lens_t = {
-  line: int,
-  signature: string,
+  sl: int,
```

Comment:
I think it'd be really good to add a comment here that explains what each of these fields are.

Reply:
Comments added.